README.md

```diff
@@ -3,69 +3,34 @@ Firefox Accounts DB Server
 
 [![Build Status](https://travis-ci.org/mozilla/fxa-auth-db-server.svg?branch=master)](https://travis-ci.org/mozilla/fxa-auth-db-server)
 
-This project implements a DB backend using MySql. It is accessible via HTTP which the
-[fxa-auth-server](https://github.com/mozilla/fxa-auth-server/) will use. Currently a work in progress.
+## Usage
 
-Note: Since this project is a work in progress, these instructions are also a WIP.
+```js
+var createServer = require('fxa-auth-db-server')
+var database = require('./db') // An object the implements the DB API described below
 
-## Prerequisites
+var server = createServer(db)
 
-* node 0.10.x or higher
-* npm
-* mysql
+server.listen(8080, 'localhost')
 
-## Configuration ##
-
-In `config.js` you can see a set of defaults for various config options. Go take a look and
-then create a new local file called `.fxa_dbrc`. This will contain a set of values to override
-the defaults. For example, if you have a password set for your MySql `root` user, you might try
-something like this:
-
-```json
-{
-"master": {
-    "password": "mysecret"
-  },
-  "slave": {
-    "password": "mysecret"
+server.on(
+  'success',
+  function (data) {
+    console.log('+ %s %s took %dms', data.method, data.url, data.t)
   }
-}
-```
-
-## Creating the Database ##
-
-Once you have your config in place, you can create and patch the database using the
-`db_patcher.js` command. Try this:
-
-```sh
-node bin/db_patcher.js
-```
-
-This should create the database (if it doesn't yet exist), and apply each patch located
-in `db/schema/*.sql` in the correct order. If this command fails and can't connect to the
-database, please check your mysql configuration and connectivity on the command line.
+)
 
-## Starting the Server ##
-
-Once the database has been created and patched, you can start the server:
-
-```sh
-npm start
+server.on(
+  'failure',
+  function (data) {
+    console.warn('- %s %s failed with %d', data.method, data.url, data.err.code)
+  }
+)
 ```
 
-Once this has started up, it will be listening on `locahost:8000` (or whatever port you
-set in your `.fxa_dbrc`..
-
-## Cleanup
-
-You may want to clear the data from the database periodically. You can just drop the database
-but make sure there is nothing in it that you want to keep:
-
-```sh
-mysql -u root -p -e 'DROP DATABASE fxa'
-```
+## DB API
 
-The server will automatically re-create it on next use.
+(todo)
```

Comment:
I don't want to block on this, but let's not delay writing these docs for too much longer